lib/macro.h

```diff
@@ -553,8 +553,14 @@ int out_len = 0;\
         _06_08_ENCODE_DROPS_FEE            (fee_ptr, fee                            );                               \
     }
 
-
-
 #endif
 
+#define OP_HOOK_INSTALLATION 1
```

Comment:
Let's move these to evernode.h because this file is directly taken from xrpld-hooks repo